### PR TITLE
Final detail tests

### PR DESCRIPTION
**Pull request by AlexandraMoga**
Final set of tests for the add-on detail page. I will add more in the future if there's need for more coverage, but now I need to start focusing on other sections.

### CODE REVIEW

File: tests/test_addon_detail.py [hunk 1]
```diff
@@ -495,3 +495,81 @@ def test_click_addon_in_more_addons_by_author(selenium, base_url, variables):
...
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    assert 'About this extension' in addon.description.addon_description_header
+    assert addon.description.addon_description_text.is_displayed()
+
+
+@pytest.mark.nondestructive
+def test_addon_ratings_card(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get('{}/addon/{}'.format(base_url, extension))
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:512]: This looks like it could be in f string format

File: tests/test_addon_detail.py [hunk 2]
```diff
@@ -495,3 +495,81 @@ def test_click_addon_in_more_addons_by_author(selenium, base_url, variables):
...
+
+@pytest.mark.nondestructive
+def test_click_addon_recommendations(selenium, base_url, variables):
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    recommendation_name = addon.recommendations.recommendations_results_item[0].text
+    # clicks on a recommendations and checks that the addon detail page is loaded
+    addon.recommendations.recommendations_results_item[0].click()
+    addon.wait_for_page_to_load()
```
Comment (jrbenny35) [inline tests/test_addon_detail.py:542]: So this waits for the same page to be loaded? Or a new page

**Review by jrbenny35 — CHANGES_REQUESTED** [review]
Just some thoughts

File: tests/test_addon_detail.py (hunk 1 above)
Comment (AlexandraMoga) [reply tests/test_addon_detail.py:512]: yeah, I don't know where I've lost the 'f'. I'll fix that.

File: tests/test_addon_detail.py (hunk 2 above)
Comment (AlexandraMoga) [reply tests/test_addon_detail.py:542]: This waits for a new add-on page to be loaded - the add-on accessed from the recommendations card.

Comment (jrbenny35) [reply tests/test_addon_detail.py:542]: that new page is still a detail page?

Comment (AlexandraMoga) [reply tests/test_addon_detail.py:542]: @jrbenny35 yes, it is another detail page

**Review by jrbenny35 — APPROVED** [review]

**Pull request merged**